README.md

```diff
@@ -1,9 +1,12 @@
 # ZKSYNC-EIGENDA M1
```

Comment:
change this title to something more descriptive, something like `Zksync-EigenDA proving sidecar`